### PR TITLE
Migrate to Awestruct 0.5.5

### PR DESCRIPTION
**Pull request by emmanuelbernard**
The upgrade went smooth. @hferentschik 
I only found some issue with our 404 page that I fixed. The rest looked like the previous site.

One question I have is on the best practice. Should we set versions of dependencies on `Gemfile`, or let `Gemfile.lock` be the version guardian. Of course the main dependencies like `awestruct` would have their version set in `Gemfile`

### CODE REVIEW

**Comment by emmanuelbernard** [comment]
Fuck you GitHub. Leave this PR open. @hferentschik pretend it is opened. I simply pushed the branch to staging for testing.

**Comment by emmanuelbernard** [comment]
Looks like there is a problem

`rake test`

```
/var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/formatters.rb:167:in `find_formatter': Formatter 'nested' unknown - maybe you meant 'documentation' or 'progress'?. (ArgumentError)
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/formatters.rb:126:in `add'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/configuration.rb:625:in `add_formatter'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/configuration_options.rb:106:in `block in load_formatters_into'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/configuration_options.rb:106:in `each'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/configuration_options.rb:106:in `load_formatters_into'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/configuration_options.rb:24:in `configure'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:96:in `setup'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:85:in `run'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:70:in `run'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/lib/rspec/core/runner.rb:38:in `invoke'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/gems/rspec-core-3.0.4/exe/rspec:4:in `<top (required)>'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/bin/rspec:19:in `load'
    from /var/lib/jenkins/jobs/staging.hibernate.org/workspace/.bundle/ruby/1.9.1/bin/rspec:19:in `<main>'
```

I'm too tired to look at that tonight. Feel free @hferentschik

**Comment by emmanuelbernard** [comment]
Looks like a rspec migration work https://github.com/cmertz/logstash_rails/issues/3 @hferentschik

**Comment by hferentschik** [comment]
I have a look today ...

**Comment by hferentschik** [comment]
I think I will create a new branch and pull request

File: Gemfile [hunk 1]
```diff
@@ -1,16 +1,21 @@
 source 'https://rubygems.org'
 
 # Main site dependencies
-gem 'awestruct', '0.5.4.rc3', :git => 'https://github.com/awestruct/awestruct', :tag => 'v0.5.4.rc3'
-gem 'uglifier', '~> 2.0.1'       # Ruby wrapper for UglifyJS JavaScript compressor
-gem 'cssminify', '~> 1.0.2'      # CSS compression using YUI compressor
-gem 'rb-fsevent', '~> 0.9.3'     # FSEvents API with Signals catching (without RubyCocoa)
-gem 'kramdown', '~> 1.0.1'       # Kramdown works on all platforms, rdiscount only on mri
-gem 'htmlcompressor', '~> 0.0.3' # Adds in HTML minification, helps remove the warning on awestruct startup
+#gem 'awestruct', '0.5.4.rc3', :git => 'https://github.com/awestruct/awestruct', :tag => 'v0.5.4.rc3'
```
Comment (hferentschik) [inline Gemfile:4]: Seems there is quite a change of gems involved. Did you check the existing list against the latest awestruct I assume, right?

Btw, I think it is the right thing to remove the versions in the Gemfile. They will get locked down in Gemfile.lock.

**Pull request merged**